nix/scripts/list-helm-containers.sh

```diff
@@ -29,8 +29,12 @@ function optionally_complain() {
 # render the charts, and assemble the list of images this would fetch.
 while IFS= read -r chart; do
   echo "Running helm template on chart ${chart}…" >&2
-
+  # The sed command removes the digest from images that have BOTH a tag and a digest, as skopeo doesn't support that currently.
```

Comment:
This comment about `sed` can be removed. That command is now in `create-container-dump`